### PR TITLE
Add authentication environments

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**
Allows for easier switching between projects. URLs for using the test examples now have the form

```
https://localhost:3000/migration?env=europewest11-3d&project=3ddemo&modelId=2890599736800729&revisionId=8160779262643447
https://localhost:3000/migration?env=greenfield-3d&project=3ddemo&modelId=4304523256471874&revisionId=1692652455232986
```

### CODE REVIEW

File: examples/src/routes.tsx [hunk 1]
```diff
@@ -69,15 +71,17 @@ export const exampleRoutes: Array<ExampleRoute> = [
     name: 'default',
     path:
       `/migration?project=${project}` +
+      `&env=${defaultEnvironmentParam}` +
       `&modelId=${cadId}` +
       `&revisionId=${cadRevisionId}`,
-    menuTitle: 'Default',
+    menuTitle: 'Abandon all hope ye who enter here',
```
Comment (larsmoa) [inline examples/src/routes.tsx:77]: wat

Comment (larsmoa) [inline examples/src/routes.tsx:77]: wat

**Review by larsmoa — CHANGES_REQUESTED** [review]
Move stuff from .env to .env.example - lets leave .env a user-defined thing.

Also, add some details on how to use .env to setup environments in README.md

File: examples/src/routes.tsx (hunk 1 above)
Comment (haakonflatval-cognite) [reply examples/src/routes.tsx:77]: Damnit, hoped you wouldn't notice :sweat_smile:

**Comment by larsmoa** [comment]
> Move stuff from .env to .env.example - lets leave .env a user-defined thing.
> 
> Also, add some details on how to use .env to setup environments in README.md

Bump @haakonflatval-cognite 

Also, update draft title before merge

**Review by larsmoa — APPROVED** [review]

**Pull request merged**